src/v3/core/graph.js

```diff
@@ -26,14 +27,23 @@ export type NeighborsOptions = {|
 export opaque type GraphJSON = any; // TODO
 
 export class Graph {
+  // A node `n` is in the graph if `_nodes.has(n)`.
+  //
+  // An edge `e` is in the graph if `_edges.get(e.address)`
+  // is deep-equal to `e`.
+  //
+  // Invariant: For an edge `e`, the following are equivalent:
```

Comment:
Note that this invariant is knowingly violated in this commit (we do not
use `_inEdges`/`_outEdges` because we do not yet have `neighbors`).

We could remove this invariant, add a note that it’s intentionally
violated in this commit, or leave it as is (my preferences in that
order). Thoughts?

Reply:
Removed.